src/crypto/crypto.ts

```diff
@@ -29,6 +42,30 @@ const ENCRYPTED_CHARS = /[a-f0-9]/;
 const KEY_NAME = 'sfdx';
 const ACCOUNT = 'local';
 
+let cryptoLogger: Logger;
+const getCryptoLogger = (): Logger => {
+  if (!cryptoLogger) {
+    cryptoLogger = Logger.childFromRoot('crypto');
+  }
+  return cryptoLogger;
+};
+
+type CryptoVersion = 'v1' | 'v2';
+let cryptoVersion: CryptoVersion;
```

Comment:
you can derive this type from (existing code `keyof typeof IV_BYTES`) or using the proposed `VERSIONS` structure, `keyof typeof VERSIONS`